Wangscape/TilesetGenerator.cpp

```diff
@@ -28,34 +28,21 @@ void TilesetGenerator::generate(std::function<void(const sf::Texture&, std::stri
     boost::filesystem::path p(options.relativeOutputDirectory);
     for (const auto& clique : options.cliques)
     {
-        size_t res_x = options.resolution;
-        size_t res_y = options.resolution;
-        bool x_dimension = true;
-        for (int i = 0; i < CORNERS; i++)
-        {
-            size_t& res_z = x_dimension ? res_x : res_y;
-            res_z *= clique.size();
-            x_dimension = !x_dimension;
-        }
```

Comment:
I agree that code needed some simplification. However, I would prefer to keep it dependent on `CORNERS` to reduce refactoring work in the future. https://github.com/hryniuk/Wangscape/pull/1/commits/bc790b57dcc4cdde19d62069da2be5141c79c830 and https://github.com/hryniuk/Wangscape/pull/1/commits/d8187380c2f896ae72ce0517d432da33f9fa2f85 take care of that. Unfortunately I committed them to the wrong branch :sweat_smile: